.github/workflows/release-swagger-ui-packagist.yml

```
@@ -0,0 +1,21 @@
+name: Release SwaggerUI to Packagist
+
+on:
+  workflow_run:
+    workflows: ["Release SwaggerUI"]
+    types:
+      - completed
+    branches: [ master ]
+
+jobs:
+  release-swagger-ui-dist:
```

Should the job name be different?
```suggestion
  release-swagger-ui-packagist:
```

---

it's arbitrary, but better to have it right. thanks!